packages/vscode/src/index.ts

```diff
@@ -35,6 +35,16 @@ export async function activate(ext: ExtensionContext) {
         files: 'nuxt.config',
         fields: 'unocss',
       }),
+      sourceObjectFields({
+        files: 'aleph.config',
+        extensions: ['js', 'mjs', 'jsx', 'ts', 'tsx'],
+        fields: 'serve/config/unocss',
+      }),
+      sourceObjectFields({
+        files: 'deno',
+        extensions: ['json', 'jsonc'],
+        fields: 'serve/config/unocss',
```

Comment:
Those fields looks a bit strange, what's the configure file would look like?

Reply:
I referenced this
https://github.com/alephjs/aleph.js/blob/main/server/helpers.ts